pf/Makefile

```diff
@@ -67,6 +67,9 @@ tidy::
 	cd tests/internal/tlsshim && go mod tidy
 	cd tests/testdatagen/genrandom && go mod tidy
 
-.PHONY: lint
+.PHONY: lint todos
 lint:
 	golangci-lint run
+
+todos::
+	git grep TODO | grep -v terraform-bridge | grep -v context.TODO
```

Comment:
I don't think we need a permanent target for this. We aren't going to use it, are we?